docs/coffee.ts

```diff
@@ -0,0 +1,273 @@
+const coffeeSelector = (document.getElementById("coffeeSelector") as HTMLInputElement);
+const teaSelector = (document.getElementById("teaSelector") as HTMLInputElement);
+const dessertSelector = (document.getElementById("dessertSelector") as HTMLInputElement);
+const menuGrid:any = document.getElementById("menuGrid");
+const menuButton = document.getElementById("coffeeMenu");
+const navModal = document.getElementById("navModal");
+let total:number;
+let products:any;
+let productsLength:number;
+
+// reading the material.JSON file
+async function fetchData() {
+  try {
+    const response = await fetch("./material.json");
+    if (!response.ok) {
+      throw new Error("could not fetch data");
+    }
+    products = await response.json();
+    productsLength = products.length;
+    if (!coffeeSelector) {throw new Error("coffee selector item-value is null")}
+    createMenuGrid(coffeeSelector.value);
+  } catch (error) {
+    console.error("Failed to fetch data", error);
+  }
+}
+
+fetchData();
+
+
+// script for creating grids
+function createMenuGrid(x:string) {
```

Comment:
Are you using Eslint or prettier? Its missing code formatting

Reply:
I'm using prettier at the moment but I have installed the ESlint extention as well. I'll check the issue and fix it.